lib/msal-common/test/request/RequestParameterBuilder.spec.ts

```diff
@@ -230,6 +230,218 @@ describe("RequestParameterBuilder unit tests", () => {
         ).toBe(true);
     });
 
+    it("Doesn't encode extra params if encodeParams is false and extra params are passed in", () => {
```

Comment:
Please add a test for a positive scenario as well

Reply:
Added